cloudflare_ddns_src/customerrors.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+
+class FetchExternalIPError(Exception):
```

Comment:
This doesn't seem very necessary to me. Why do we need a dedicated exception class?

This also introduces additional overhead needing to install and include an addditional directory `cloudflare_ddns_src`; I'd rather everything be contained in a single file.

Reply:
I used the custom exceptions to handle errors more easily at [this line](https://github.com/LINKIWI/cloudflare-ddns-client/pull/35/files#diff-313c56cbe7ec120d363492f37306ec95R288). Also see [here](https://stackoverflow.com/a/417457/9850709).

However, I wasn't sure about introducing new files either. 
It provides better overview of your souce code and it should be safe installed by your makefile.
On the other hand, I also see the benefits of one-file projects.

My aim was to provide detailed information, so noone has to check the source code to understand what is failing. But I will change it as you want to :)